src/elements/hx-toggle/index.spec.js

```diff
@@ -0,0 +1,121 @@
+            it('markup should contain a #hxToggle <div>', async () => {
+                const elSelector = 'div#hxToggle';
+                const id = 'hxToggle';
+                const component = /** @type { HXToggleElement } */ await fixture(template);
+                const shadow = component.shadowRoot;
+                const query = shadow.querySelector(elSelector);
+                const queryId = query.id;
+
+                expect(queryId).to.equal(id);
+            });
+
+            it('markup should contain a #hxToggleGrid <div>', async () => {
+                const elSelector = 'div#hxToggle > #hxToggleGrid';
+                const id = 'hxToggleGrid';
+                const component = /** @type { HXToggleElement } */ await fixture(template);
+                const shadow = component.shadowRoot;
+                const query = shadow.querySelector(elSelector);
+                const queryId = query.id;
+
+                expect(queryId).to.equal(id);
+            });
+
+            it('markup should contain grid type <hx-icon>', async () => {
+                const elSelector = 'div#hxToggleGrid > hx-icon';
+                const selectType = 'grid';
+                const component = /** @type { HXToggleElement } */ await fixture(template);
+                const shadow = component.shadowRoot;
+                const query = shadow.querySelector(elSelector);
+                const queryType = query.type;
+
+                expect(queryType).to.equal(selectType);
+            });
+
+            it('markup should contain a #hxToggleList <div>', async () => {
+                const elSelector = 'div#hxToggle > #hxToggleList';
+                const id = 'hxToggleList';
+                const component = /** @type { HXToggleElement } */ await fixture(template);
+                const shadow = component.shadowRoot;
+                const query = shadow.querySelector(elSelector);
+                const queryId = query.id;
+
+                expect(queryId).to.equal(id);
+            });
+
+            it('markup should contain list type <hx-icon>', async () => {
+                const elSelector = 'div#hxToggleList > hx-icon';
+                const selectType = 'list';
+                const component = /** @type { HXToggleElement } */ await fixture(template);
+                const shadow = component.shadowRoot;
+                const query = shadow.querySelector(elSelector);
+                const queryType = query.type;
+
+                expect(queryType).to.equal(selectType);
+            });
+        });
+    });
+
+    describe('test <hx-toggle> getter and setter methods', () => {
+        it('should have toggled attribute', async () => {
+            const fragment = /** @type {HXToggleElement} */ await fixture(mockup);
+            const attr = fragment.hasAttribute('toggled');
+            fragment.toggled = true;
+            const checkattr = fragment.hasAttribute('toggled');
```

Comment:
Please change variable to `toggledAttr`.

Reply:
Changes have been done, please review.